tasks/pretest.js

```diff
@@ -52,6 +53,16 @@ function makeTestImageFolders() {
     makeOne(constants.pathToTestImagesDiff, 'test image diff folder');
 }
 
+// Make script file that define plotly in a RequireJS context
+function makeRequireJSFixture() {
+    var bundle = fs.readFileSync(constants.pathToPlotlyDistMin, 'utf-8'),
+        template = 'define(\'plotly\', function(require, exports, module) { {{bundle}} });',
+        index = template.replace('{{bundle}}', bundle);
+
+    common.writeFile(constants.pathToRequireJSFixture, index);
```

Comment:
this basically mimics how plotly.py inject `plotly.min.js` - see [here](https://github.com/plotly/plotly.py/blob/22cb9d92bb8485f660aab60cc58ea1791505a097/plotly/offline/offline.py#L147-L157).